internal/build/tar.go

```diff
@@ -120,6 +120,9 @@ type archiveEntry struct {
 // e.g. tarring my_dir --> dest d: d/file_a, d/file_b
 // If source path does not exist, quietly skips it and returns no err
 func (a *ArchiveBuilder) entriesForPath(ctx context.Context, source, dest string) ([]archiveEntry, error) {
+	// (PL) convert \ to / in case of windows path
+	source = filepath.ToSlash(source)
```

Comment:
should there be separate variables for `sourceAsHostOS` and `sourceInContainer`? i want to make sure the `os.Stat` call below works correctly on Windows, and i think converting to slash here might make it not work right?

Reply:
Hi @nicks

AFAIK Windows will accept path that are in the form c:/xxxx/dsdsd, so just to get a common ground between Linux and Windows i transform everything to forward slash, but if you want to use separate variables there no problems either